spring-cloud-config-server/src/main/java/org/springframework/cloud/config/server/environment/JGitEnvironmentRepository.java

```diff
@@ -307,6 +341,17 @@ private FetchResult fetch(Git git, String label) {
 			}
 			return result;
 		}
+		catch (TransportException ex){
+			if(errorLogOnFailingFetch){
```

Comment:
I don't know. Is a `TransportException` thrown if a repository doesn't exist? That is a regular occurrence that isn't an error.